include/openthread/platform/radio.h

```diff
@@ -798,6 +829,18 @@ extern void otPlatRadioReceiveDone(otInstance *aInstance, otRadioFrame *aFrame,
  */
 extern void otPlatDiagRadioReceiveDone(otInstance *aInstance, otRadioFrame *aFrame, otError aError);
 
+/**
+ * The radio driver completed the interface switchover procedure
+ *
+ * @param[in]  aInstance The OpenThread instance structure.
+ * @param[in]  aFrame    A pointer to the received frame or NULL if the receive operation failed.
+ * @param[in]  aError    OT_ERROR_NONE when successfully received a frame,
+ *                       OT_ERROR_ABORT when reception was aborted and a frame was not received,
+ *                       OT_ERROR_NO_BUFS when a frame could not be received due to lack of rx buffer space.
+ *
+ */
+extern void otPlatRadioSwitchoverDone(otInstance *aInstance, bool success);
```

Comment:
What is the use case for this "switch-over" callback?

Reply:
This is related to otPlatSetMultipanInterface and is intended to notify host that active interface has changed. May be needed if platform requires more time to switch interfaces or when aCompletePending is set to true and interfaces will be switched after ongoing tx/rx operation is complete.

This could be used to invoke otPlatSetMultipanInterface(...aCompletePending=false) if otPlatRadioSwitchoverDone is not called in time after otPlatSetMultipanInterface(...aCompletePending=true), or just to measure switching time.